spec/rspec/rails/matchers/have_rendered_spec.rb

```diff
@@ -88,6 +88,24 @@ def assert_template(*); raise "oops"; end
           end.to raise_exception("oops")
         end
       end
+
+      context "when fails with a redirect" do
+        let(:response) { ActionController::TestResponse.new(302) }
+        def assert_template(*)
+          message = "expecting <'template_name'> but rendering with <[]>"
+          raise ActiveSupport::TestCase::Assertion.new(message)
+        end
+        def normalize_argument_to_redirection(*)
+          "http://test.host/widgets/1"
+        end
```

Comment:
Instead of `def normalize_argument_to_redirection(*)` could we update this to `def normalize_argument_to_redirection(_descriptive_name_of_what_this_is`?